### DISCUSSION TITLE
Does winmerge compare lines out of order?

### DISCUSSION THREAD

**Discussion by dat-bingham99** [WinMerge/winmerge#2353] [Q&A] [answered]
Is winmerge able to compare files where they are not in the same order?  i.e. File content is same but ordering is not.

**Comment by sdottaka** [comment, marked as answer]
WinMerge does not have the ability to consider files with different line sequences to be matching.
Use the SortAsceding or SortDescending plugin to sort and compare files as follows:

![image](https://github.com/WinMerge/winmerge/assets/98126/f5689343-92a3-4ffa-9e82-5a34d9dd30f1)
![image](https://github.com/WinMerge/winmerge/assets/98126/655f42e3-f331-4b4c-955d-391356c8043a)
